x/concentrated-liquidity/lp.go

```diff
@@ -8,24 +8,36 @@ import (
 	types "github.com/osmosis-labs/osmosis/v12/x/concentrated-liquidity/types"
 )
 
-func (k Keeper) Mint(ctx sdk.Context, poolId uint64, owner sdk.AccAddress, tokenIn sdk.Int, lowerTick sdk.Int, upperTick sdk.Int) (numShares sdk.Int, err error) {
-	// first check and validate arguments
+func (k Keeper) Mint(ctx sdk.Context, poolId uint64, owner sdk.AccAddress, liquidityIn sdk.Int, lowerTick sdk.Int, upperTick sdk.Int) (amtDenom0, amtDenom1 sdk.Int, err error) {
 	if lowerTick.GTE(types.MaxTick) || lowerTick.LT(types.MinTick) || upperTick.GT(types.MaxTick) {
-		// TODO: come back to errors
-		return sdk.Int{}, fmt.Errorf("validation fail")
+		return sdk.Int{}, sdk.Int{}, fmt.Errorf("validation fail")
 	}
 
-	if tokenIn.IsZero() {
-		return sdk.Int{}, fmt.Errorf("token in amount is zero")
+	if liquidityIn.IsZero() {
+		return sdk.Int{}, sdk.Int{}, fmt.Errorf("token in amount is zero")
 	}
 
-	k.UpdateTickWithNewLiquidity(ctx, poolId, lowerTick, tokenIn)
-	k.UpdateTickWithNewLiquidity(ctx, poolId, upperTick, tokenIn)
+	// k.UpdateTickWithNewLiquidity(ctx, poolId, lowerTick, liquidityIn)
+	// k.UpdateTickWithNewLiquidity(ctx, poolId, upperTick, liquidityIn)
 
-	// update tick with new liquidity
-	k.updatePositionWithLiquidity(ctx, poolId, owner.String(), lowerTick, upperTick, tokenIn)
+	// k.updatePositionWithLiquidity(ctx, poolId, owner.String(), lowerTick, upperTick, liquidityIn)
```

Comment:
is this going to get reintroduced?

Reply:
this is commented out bc we do not have code for initialization